Mada_Alwuthaynani/week_01/day_02/main.js

```diff
@@ -0,0 +1,54 @@
+var numOne=1;
+var numTwo=2;
+if(numOne>numTwo){
+    console.log(numOne);}
+    else if(numOne<numTwo){
+        console.log(numTwo);  
+    }
+else{console.log("equale");}
+var age=18;
+var wait=0;
+if(age<18){
+    wait=18-age;
+    console.log("Drive away! "+wait+" years to wait until you can drive"); }
```

Comment:
Great work Mada!
This is probably just a copy/paste typo, but the sentence should say `Sorry, you have X years to...`